functions/src/models/offers/index.ts

```diff
@@ -1,5 +1,6 @@
+import { FirestoreDataConverter } from '@google-cloud/firestore';
 import { IsEnum, IsNotEmpty, IsObject, IsString, ValidateNested } from 'class-validator';
-import { firestore } from 'firebase';
+import { firestore } from 'firebase-admin';
```

Comment:
Is there a reason for this change?

This code should have been working without this change.

Reply:
yea, so the `firebase` package is for the client-side, and we are using `firebase-admin` in the cloud function, right? so I thought of keeping it the same as the cloud function.
We can change it back